Sources/Widgets/Widgets3D/LineWidget/index.js

```diff
@@ -108,15 +116,20 @@ function vtkLineWidget(publicAPI, model) {
             initialValues: {
               /* to scale handle size when zooming/dezooming, optionnal */
               scaleInPixels: true,
-              /* to detect arrow type in ArrowHandleRepresentation, mandatory */
+              /* to detect arrow       // manageVoidHandles();type in ArrowHandleRepresentation, mandatory */
               handleType: model.handle2Shape,
+              /* To detect in handleRepresentation the origin of LineWidget, mandatory */
+              fromLineWidget: true,
+              handleVisibility: model.handle2Visibility,
+              handleCameraOrientation: model.handle2CameraOrientation,
             },
           },
           {
             builder: vtkSVGLandmarkRepresentation,
             initialValues: {
               showCircle: false,
               isVisible: false,
```

Comment:
is it really useful to set that variable?  I don't see where it is being used

Reply:
all of these variables operate on the handleRepresentation level

Reply:
I do not see where `isVisible` is being used